src/Dialog/Dialog.js

```diff
@@ -46,6 +45,92 @@ export const styleSheet = createStyleSheet('MuiDialog', (theme) => {
   };
 });
 
+type Props = {
```

Comment:
Hold on with the flow migration, I'm refactoring all the components to use `withStyles`. Let's try to avoid conflict 🙈 .